tree/tree.go

```diff
@@ -0,0 +1,131 @@
+package tree
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"hash/fnv"
+	"io/ioutil"
+	"sort"
+
+	"github.com/treeverse/lakefs/tree/sstable"
+)
+
+const SplitFactor = 200_000 // avarage number of entries in a part file. used to compute modulo on hash of path
+
+type TreePartType struct {
+	PartName string `json:"part_name"`
```

Comment:
why json?

Reply:
BTW - this file belongs in my implementation of the tree. should be removed